x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/connector_edit_flyout.scss

```
@@ -1,3 +1,3 @@
 .connectorEditFlyoutTabs {
-  margin-bottom: '-25px';
+  margin-bottom: -25px;
```

Use EUI variables whenever possible, `euiSizeL` at 24px should be a good replacement.
```suggestion
  margin-bottom: -$euiSizeL;
```

---

Cool, I didn't know about those!
Thanks @andreadelrio